pkg/grpc/client.go

```diff
@@ -0,0 +1,76 @@
+package grpc
```

Comment:
`pkg/grpc` is going to be a public pkg in the CLI that everyone will be able to consume. This implementation exposes a very specific codespaces implementation. What do you think of `liveshare/grpc` as the location for the package?

Reply:
Ah I didn't realize that the packages are public, thanks for letting me know. My only concern with having it in `liveshare/grpc` is sharing it with the new connection stack later since we do want it to independent. Could we have it in `internal/grpc` instead?

Reply:
Perhaps `internal/codespaces/grpc`?

Reply:
Works for me!